.gitmodules

```diff
@@ -2,56 +2,44 @@
 	path = binutils
 	url = https://sourceware.org/git/binutils-gdb.git
 	branch = binutils-2_43-branch
-	shallow = true
 [submodule "gcc"]
 	path = gcc
 	url = https://gcc.gnu.org/git/gcc.git
 	branch = releases/gcc-14
-	shallow = true
 [submodule "glibc"]
 	path = glibc
 	url = https://sourceware.org/git/glibc.git
-	shallow = true
 [submodule "dejagnu"]
 	path = dejagnu
 	url = https://git.savannah.gnu.org/git/dejagnu.git
 	branch = master
-	shallow = true
 [submodule "newlib"]
 	path = newlib
 	url = https://sourceware.org/git/newlib-cygwin.git
 	branch = master
-	shallow = true
 [submodule "gdb"]
 	path = gdb
 	url = https://sourceware.org/git/binutils-gdb.git
 	branch = gdb-15-branch
-	shallow = true
 [submodule "qemu"]
 	path = qemu
 	url = https://gitlab.com/qemu-project/qemu.git
-	shallow = true
 [submodule "musl"]
 	path = musl
 	url = https://git.musl-libc.org/git/musl
 	branch = master
-	shallow = true
 [submodule "spike"]
 	path = spike
 	url = https://github.com/riscv-software-src/riscv-isa-sim.git
 	branch = master
-	shallow = true
 [submodule "pk"]
 	path = pk
 	url = https://github.com/riscv-software-src/riscv-pk.git
 	branch = master
-	shallow = true
 [submodule "llvm"]
 	path = llvm
 	url = https://github.com/llvm/llvm-project.git
-	branch = release/18.x
-	shallow = true
+	branch = release/19.x
```

Comment:
Branch `release/19.x`:

- https://github.com/llvm/llvm-project/tree/release/19.x

is commit `61c9f97`:

- [61c9f97](https://github.com/llvm/llvm-project/commit/61c9f970b88710c8292cc32b92c1db723fcfff22)

But this PR changes the submodule commit from `3b5b5c` to `e21dc4b`.

Isn't this a mismatch?

Reply:
The branch `release/19.x` has tag `llvmorg-19.1.6` ( e21dc4b, https://github.com/llvm/llvm-project/tree/llvmorg-19.1.6 ) . When I submitted (including now), there was no `llvmorg-19.1.7` (61c9f97) related tag. I will update it as soon as it becomes available.